python/gink/impl/memory_store.py

```diff
@@ -62,6 +62,15 @@ def __init__(self) -> None:
         self._verify_keys = dict()
         self._logger = getLogger(self.__class__.__name__)
 
+    def drop_history(self, as_of: Optional[MuTimestamp] = None):
+        raise NotImplementedError("MemoryStore.drop_history is not implemented")
```

Comment:
Please add an issue to implement history management in the memory story if you haven't already.
In any case, it would be good for us to link to the issue in a comment or in the exception whenever we
have a not implemented path.